examples/conway/src/main.rs

```diff
@@ -31,7 +31,9 @@ fn main() -> Result<(), Error> {
         if let Event::RedrawRequested(_) = event {
             life.draw(pixels.get_frame());
             if pixels
-                .render()
+                .render(|encoder, render_target, scaling_renderer| {
+                    scaling_renderer.render(encoder, render_target);
+                })
```

Comment:
Can we simplify this to `pixels.render(None)`?

Reply:
As in make render() take Option<FnOnce>, with None corresponding to this? Sure.